include/functions.h

```diff
@@ -408,12 +408,12 @@ void func_8002F758(PlayState* play, Actor* actor, f32 arg2, s16 arg3, f32 arg4,
 void func_8002F7A0(PlayState* play, Actor* actor, f32 arg2, s16 arg3, f32 arg4);
 void Player_PlaySfx(Player* player, u16 sfxId);
 void Actor_PlaySfx(Actor* actor, u16 sfxId);
-void func_8002F850(PlayState* play, Actor* actor);
-void func_8002F8F0(Actor* actor, u16 sfxId);
-void func_8002F91C(Actor* actor, u16 sfxId);
-void func_8002F948(Actor* actor, u16 sfxId);
-void func_8002F974(Actor* actor, u16 sfxId);
-void func_8002F994(Actor* actor, s32 timer);
+void Actor_PlaySfx_SurfaceBomb(PlayState* play, Actor* actor);
+void Actor_PlaySfx_Flagged2(Actor* actor, u16 sfxId);
+void Actor_PlaySfx_FlaggedCentered1(Actor* actor, u16 sfxId);
+void Actor_PlaySfx_FlaggedCentered2(Actor* actor, u16 sfxId);
+void Actor_PlaySfx_Flagged(Actor* actor, u16 sfxId);
+void Actor_PlaySfx_FlaggedTimer(Actor* actor, s32 timer);
```

Comment:
1) Im not sure I like "flagged" but dont have suggestions atm
2) I would prefer `Actor_PlayXyzSfx` instead of a subsytem
I get the point of this is MM sync, so dont have to change now but just putting my thoughts out

Reply:
I'm open to renaming and porting the renames to MM too but yeah idk what to name these

Reply:
yeah ill have a look at it soon